### PR TITLE
Refactoring of actions to use plain fieldnames and separate fqid in the action

### PR DESCRIPTION
**Pull request by ThomasJunk**
I looked into #28 and talked to @FinnStutzenstein about the current and future structure. 
And I came up with this changeset.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,4 +46,3 @@ To start it run
 * OPENSLIDES_BACKEND_EVENT_STORE_URL
 
   URL of event store service. Default: http://localhost:9003/
-n
```
Comment (normanjaeckel) [inline README.md]: ??? Maybe just remove it.

File: tests/actions/test_committee.py [hunk 2]
```diff
@@ -28,7 +28,8 @@ class CommitteeCreateActionUnitTester(BaseCommitteeCreateActionTester):
     def setUp(self) -> None:
         super().setUp()
         self.action = CommitteeCreate(PermissionTestAdapter(), DatabaseTestAdapter())
-        self.action.user_id = 7668157706  # This user has perm COMMITTEE_CAN_MANAGE.
+        # This user has perm COMMITTEE_CAN_MANAGE.
```
Comment (normanjaeckel) [inline tests/actions/test_committee.py]: Please revert this change. It is much easier for me to copy past a user_id with its short description in test code, if it remains in one line. Please revert also in all other cases.

File: tests/utils.py [hunk 3]
```diff
@@ -69,7 +69,7 @@ def get_fqid(value: str) -> FullQualifiedId:
     """
     Returns a FullQualifiedId parsed from the given value.
     """
-    collection, id = value.split(KEYSEPARATOR)
+    collection, id, *catch_trailing_slash = value.split(KEYSEPARATOR)
```
Comment (normanjaeckel) [inline tests/utils.py]: Please do not catch more slashes here. Instead we should check an raise an error if someone tries to use an invalid value here.

**Comment by ThomasJunk** [comment]
I undid the changes.

**Comment by normanjaeckel** [comment]
See #35
